### PR TITLE
enhancement/966 - Improve terrain generation documentation

### LINKED ISSUE #966: Improve terrain generation documentation
The documentation is hard to follow and could be a lot clearer. A video should also be uploaded to the YouTube channel showing the process in detail (just like [this video](https://www.youtube.com/watch?v=tWky-8QWFWo) by @erikquinn)

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves openscope/openscope#966

### CODE REVIEW

**Comment by Fechulo** [comment]
It still a work in progress and subject to more changes, but I just wanted some feedback on the changes I've made so far

File: documentation/airport-terrain-generation.md [hunk 1]
```diff
@@ -1,190 +1,185 @@
...
-  Search for 'Zonal statistics plugin'
-  Ensure the plugin is checked in the middle pane
-  Click Close
+#### 4. Obtain elevation data
+* Go to [this site](http://viewfinderpanoramas.org/Coverage%20map%20viewfinderpanoramas_org3.htm) and download all the tiles which cover your airport and its surrounding area
+* Extract the zip files into a new folder `~/terrain/DEM/`
 
+#### 5. Obtain airspace coordinates
+* Copy all the coordinates within the airspace key of your airport file
+* Convert the coordinates to the right format using [this web app](https://jakcharvat.github.io/polygonConverter/) *(Thanks @jakcharvat for this wonderful tool!)*
```
Comment (Marcel510) [inline documentation/airport-terrain-generation.md]: Maybe it should be mentioned that the airspace coordinates have to be in degree-only format to use them with the web app. (Eventhough the web app has code for converting deg min sec to deg, it didn't work for me)

Comment (erikquinn) [reply documentation/airport-terrain-generation.md]: It can be in decimal degrees. If it's actually not taking that, it would be a bug. Can you reverify and if it's definitely broken, open a new issue.

Comment (Marcel510) [reply documentation/airport-terrain-generation.md]: The decimal degree format works: f.e. [41.471, 27.19383333333]
Also the degree minute second format works: f.e. ["N40d59m78", "E27d11m65"]
But it can't take degree with decimal minutes: f.e. ["N41d28.26", "E27d11.63"]

Comment (indianbhaji) [reply documentation/airport-terrain-generation.md]: @jakcharvat

Comment (jakcharvat) [reply documentation/airport-terrain-generation.md]: Ok, I'll look into that. It's probably because I was working with a degree-minute-second-decimal format ["N50d35m48.47", "E013d47m59.46"] so I'll look for a way to work with a degree-minute-decimal format as well

Comment (jakcharvat) [reply documentation/airport-terrain-generation.md]: Done. Pushed an update to the app to support the format @Marcel510 . Lemme know if there's anything else broken please

**Comment by Fechulo** [comment]
I'll keep it as a WIP for now, there's still a few details I want to change and I really think we need that video and we need it yesterday

**Comment by jakcharvat** [comment]
What if we also added a basic guide to creating the video map, explaining what it requires, how to do it, etc in this manual?

**Comment by Fechulo** [comment]
The problem with that is that the process is completely different from airport to airport. Every country does it differently and there's not just one source for all the data and sometimes there's no source at all and we just have to improvise so I don't know how we could write a comprehensive guide on that.

**Comment by jakcharvat** [comment]
Maybe not comprehensive, but basics. Something like "start by adding your runways to the map. Then use this tool (bit.ly/centrelines) (yeah, I made that) to generate the centrelines for the runways you just added. Then add in the restricted airspaces". Nothing too comprehensive or complicated, just the main stuff

File: documentation/airport-terrain-generation.md [hunk 2]
```diff
@@ -1,190 +1,202 @@
...
-Copy the airspace:poly: key from the existing airport and 
-paste it into a new file called ~/projects/openscope-terrain/airport.csv
-Edit airport.csv so it reads something like
-POLYGON((-121.69999940 47.83333330,-121.97603665
-47.95335007,-121.96666670 48.30000000))
-copy the first entry to the end as the last entry to close the polygon
-Note the latitude and longitude are reversed in order compared to the
-definition in the airport file and S/W entries are replaced with a
-minus (-), coordinates also need to be in decimal degrees.
+*Note: QGIS may seem frozen while the algirthm is run, but it is still running. Try not clicking anywhere on the prograam to ensure it does not crash.*
```
Comment (Marcel510) [inline documentation/airport-terrain-generation.md]: typo: "prograam"

Comment (sam-irl) [reply documentation/airport-terrain-generation.md]: and 'algirthm' should be 'algorithm'

**Comment by Fechulo** [comment]
@erikquinn This can be merged now. I don't want the video to keep holding this back.
